openBCIBoard.js

```diff
@@ -60,34 +66,57 @@ function OpenBCIFactory() {
         opts.simulatorSampleRate = options.simulatorSampleRate || options.simulatorsamplerate || _options.simulatorSampleRate;
         opts.baudRate = options.baudRate || options.baudrate || _options.baudrate;
         opts.verbose = options.verbose || _options.verbose;
+        opts.ntp = options.NTP || options.ntp || _options.NTP;
         // Set to global options object
         this.options = opts;
 
         /** Properties (keep alphabetical) */
         // Arrays
         this.writeOutArray = new Array(100);
+        this.channelSettingsArray = k.channelSettingsArrayInit(this.numberOfChannels());
         // Bools
         this.isLookingForKeyInBuffer = true;
         // Buffers
         this.masterBuffer = masterBufferMaker();
-        this.moneyBuf = new Buffer('$$$');
-        this.searchingBuf = this.moneyBuf;
+        this.searchBuffers = {
+            timeSyncStart: new Buffer('$a$'),
+            miscStop: new Buffer('$$$')
+        };
+        this.searchingBuf = this.searchBuffers.miscStop;
         // Objects
+        this.goertzelObject = openBCISample.goertzelNewObject(this.numberOfChannels());
         this.writer = null;
         this.impedanceTest = {
             active: false,
             isTestingPInput: false,
             isTestingNInput: false,
             onChannel: 0,
-            sampleNumber: 0
+            sampleNumber: 0,
+            continuousMode: false
+        };
+        this.sync = {
+            npt1: 0,
+            ntp2: 0
+        };
+        var ntpOptions = {
+            host: 'nist1-sj.ustiming.org',  // Defaults to pool.ntp.org
```

Comment:
is the computer also being synced. this would be necessary as well if you are syncing to a remote server IIUC